src/config/api.js

```diff
@@ -19,6 +19,7 @@ Kitsu.headers['User-Agent'] = `KitsuMobile/${kitsuConfig.version} (askar)`;
 const errorMiddleware = {
   name: 'error-middleware',
   error: (payload) => {
+    console.log('failed payload', payload);
```

Comment:
better remove this

Reply:
@edencakir there's no way to identify when there's an unknown error caught by middleware.